packages/cart/src/models/cart.ts

```diff
@@ -29,7 +29,11 @@ export default class Cart {
   @PrimaryKey({ columnType: "text" })
   id: string
 
-  @Property({ columnType: "text", nullable: true })
+  @Property({
+    columnType: "text",
+    nullable: true,
+    index: "IDX_cart_region_id",
+  })
   region_id?: string | null
```

Comment:
```suggestion
  region_id: string | null = null
```

Reply:
I will clean up all cart data models with respect to our new convention. They haven't been updated since we decided to do `= null`